src/helpers/transformUniqueAssetTraitsForPresentation.ts

```diff
@@ -0,0 +1,53 @@
+import { format } from 'date-fns';
+import { UniqueAssetTrait } from '../entities/uniqueAssets';
+
+type MappedTrait = UniqueAssetTrait & {
+  originalValue: string | number | null | undefined;
+  lowercase?: boolean;
+  disableMenu?: boolean;
+};
+
+/**
+ * Mapper function that converts various NFT trait values according to their
+ * data type and other qualities. Like if it's a link it is shortening it.
+ */
+export default function transformUniqueAssetTraitsForPresentation(
+  trait: UniqueAssetTrait
+): MappedTrait {
+  const { display_type, value } = trait;
+
+  if (display_type === 'date') {
+    // the value is in seconds with milliseconds in the decimal part
+    // formatted like Jan 29th, 2022
+    const newValue =
+      typeof value === 'number' ? format(value * 1000, 'MMM do, y') : value;
+
+    return {
+      ...trait,
+      disableMenu: true,
+      originalValue: value,
+      value: newValue,
+    };
+  }
+
+  if (display_type === 'boost_percentage') {
+    return { ...trait, originalValue: value, value: `+${value}%` };
+  }
+
+  if (display_type === 'boost_number') {
+    return { ...trait, originalValue: value, value: `+${value}` };
+  }
+
+  if (typeof value === 'string' && value.toLowerCase().startsWith('https://')) {
```

Comment:
Maybe we should also support `http://`?

Reply:
I think we should filter out `http`. This is probably scam

Reply:
We could

Reply:
I added support for opening `http://` but actually in terms of scams this is what I started thinking about. How do we filter malicious looking urls? With current context menu option anyone can send you an NFT with a link, isn't it some kind of an attack vector? Should we care?